interface/src/instruction.rs

```diff
@@ -924,7 +850,10 @@ pub fn move_lamports(
 
 #[cfg(test)]
 mod tests {
-    use {super::*, crate::instruction::InstructionError};
+    use {
+        crate::error::StakeError, solana_decode_error::DecodeError,
+        solana_instruction::error::InstructionError,
+    };
```

Comment:
nit: since these tests don't use the instructions at all, how about moving them into `error.rs`?

Reply:
Good idea.